nyaa/search.py

```diff
@@ -55,38 +63,13 @@ def description(self):
         # Description isn't included in import_to_es (sync_es appears to have it), but only views for individual torrents use it, so it's not needed here
         return None
 
-    @property
-    def magnet_uri(self):
-        return torrents.create_magnet(self)
-
-    @property
-    def info_hash(self):
-        return bytes.fromhex(self.elastic.info_hash)
-
-    @property
-    def main_category(self):
-        # TODO: Same, hitting the db 
-        return  self.sub_category.main_category
-
-    @property
-    def sub_category(self):
-        # TODO: Same, hitting the db 
-        return models.SubCategory.by_category_ids(self.elastic.main_category_id, self.elastic.sub_category_id)
-
-    @property
-    def stats(self):
-        return ElasticTorrentStatsAccessor(self.elastic)
-
 class ElasticSearchResultAccessor:
 
     def __init__(self, elasticResult):
         self.elastic = elasticResult
+        self.total = self.elastic.hits.total
 
     @property
-    def total(self):
```

Comment:
Extending an ABC, as mentioned above, would likely make this a good candidate for a `__len__` function.